examples/home/BUILD.bazel

```diff
@@ -0,0 +1,29 @@
+load("//distroless:defs.bzl", "home")
+load("//distroless/tests:asserts.bzl", "assert_tar_listing")
+
+home(
+    name = "home",
+    dirs = [
+        {
+            "home": "./root",
```

Comment:
using `./` feels odd to me, it should be at root? And this make it seems like it's not?